src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -399,6 +399,33 @@ public void TestNormalizeStatement1()
             TestNormalizeStatement("Func<string, int> f = blah;", "Func<string, int> f = blah;");
         }
 
+        [Theory]
+        [InlineData("[return:A]void Local([B]object o){}", "[return: A]\r\nvoid Local([B] object o)\r\n{\r\n}")]
```

Comment:
nit: Consider making the input worse (literally the opposite for every position where possible on the attributes):
```suggestion
        [InlineData("[ return:A ]void Local( [ B ]object o){}", "[return: A]\r\nvoid Local([B] object o)\r\n{\r\n}")]
```